libs/components/toast/src/lib/modules/toast/toaster.component.scss

```diff
@@ -1,29 +1,57 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
 @use 'libs/components/theme/src/lib/styles/variables' as *;
 
+@include compatMixins.sky-default-overrides('.sky-toaster') {
+  --sky-override-toaster-padding: 0 #{$sky-margin-double $sky-margin-double} 0;
+  --sky-override-toaster-max-width: 300px;
+}
+
+@include compatMixins.sky-modern-overrides('.sky-toaster') {
+  --sky-override-toaster-width-xs: var(--modern-size-320);
+  --sky-override-toaster-width: var(--modern-size-400);
+}
+
 .sky-toaster {
   bottom: 0;
   right: 0;
   display: block;
-  max-width: 300px;
+  max-width: var(--sky-override-toaster-max-width, none);
   max-height: 100%;
   overflow-y: auto;
   position: fixed;
-  padding-bottom: $sky-margin-double;
-  padding-right: $sky-margin-double;
+  padding: var(
+    --sky-override-toaster-padding,
+    var(--sky-space-inset-balanced-xl) var(--sky-space-inset-balanced-xl) 0
+  );
 }
 
 .sky-theme-modern {
   .sky-toaster {
-    max-width: none;
-    width: 320px;
-    padding: $sky-theme-modern-padding-even-xl;
-    padding-bottom: 0;
+    width: var(
+      --sky-override-toaster-width-xs,
+      max(
+        var(--sky-size-width-viewport-smallest),
+        calc(
+          min(var(--sky-size-max_width-overlay_message)) +
+            calc(var(--sky-space-inset-balanced-xl) * 2)
+        )
+      )
+    );
   }
 
   @include mixins.sky-host-responsive-container-sm-min(false) {
     .sky-toaster {
-      width: 400px;
+      width: var(
+        --sky-override-toaster-width,
+        max(
```

Comment:
Once we switch to V2 of modern we won't need to responsive container section here - but until then I had to specify the width calculation twice due to the override being different for the two sizes.